### PR TITLE
Add guest blog post on the power of upstream support

### PR DESCRIPTION
**Pull request by lblomenkemper**

### CODE REVIEW

**Comment by rfay** [comment]
What an awesome piece of work @lblomenkemper - thanks so much. And you did it all, making the whole thing work, without a bit of support. 

I had some weird problems with the graphic, I guess it was characters that looked like hyphens that weren't. 

I made some slight changes to the text and would appreciate your review.

A full generated preview is at https://20230827-lblomenkemper-why-d.ddev-com-front-end.pages.dev/blog/unleashing-php-development-with-ddev/

**Comment by lblomenkemper** [comment]
Thanks! I like the changes, ok to merge from my side

**Comment by rfay** [comment]
Do you mind if I change the title to "Why our agency became a major supporter of DDEV ... and yours should too" ?

**Comment by lblomenkemper** [comment]
fine with that, go ahead

**Comment by rfay** [comment]
It's live, thanks so much! https://ddev.com/blog/unleashing-php-development-with-ddev/

File: src/content/blog/unleashing-php-development-with-ddev.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Unleashing PHP Development with DDEV: The Power of Upstream Support"
+title: "Why Our Agency Became a Major Supporter of DDEV ... and Yours Should Too"
```
Comment (mattstein) [inline src/content/blog/unleashing-php-development-with-ddev.md:2]: I’m late @rfay, but dropping the `...` would strengthen this further. (For me `...` amounts to an unnecessary pause and visual interruption.)

**Pull request merged**